frontend/routes/viewerGui/components/sequences/components/sequencePlayer/sequencePlayer.component.tsx

```diff
@@ -1,346 +1,346 @@
-			this.setState({stepScale: this.props.stepScale});
-		}
-	}
-
-	public componentWillUnmount() {
-		this.stop();
-	}
-
-	public rewind = () => {
-		this.stop();
-		this.prevStep();
-	}
-
-	public forward =  () => {
-		this.stop();
-		this.nextStep();
-	}
-
-	private renderLoader = renderWhenTrue(() => (
-		<StyledLoader horizontal size={14} content="Loading frame..." />
-	));
-
-	public render() {
-		const { value, stepScale , stepInterval } = this.state;
-
-		return (
-			<SequencePlayerContainer>
-				<SequencePlayerColumn>
-					<SequenceRow>
-						<Grid item>
-							<IconButton disabled={this.isFirstDay} onClick={this.rewind} size="small">
-								<StepBackIcon fontSize="large" />
-							</IconButton>
-						</Grid>
-						<Grid item>
-							<DatePicker
-								shouldDisableDate={(date) => isDateOutsideRange(this.props.min, this.props.max, date.$d)}
-								name="date"
-								inputId="1"
-								value={value}
-								format={LONG_DATE_TIME_FORMAT_NO_MINUTES}
-								onChange={(e) => this.gotoDate(new Date(Math.floor(e.target.value / MILLI_PER_HOUR) * MILLI_PER_HOUR))}
-								placeholder="date"
-								dateTime
-							/>
-						</Grid>
-						<Grid item>
-							<IconButton disabled={this.isLastDay} onClick={this.forward} size="small">
-								<StepForwardIcon fontSize="large" />
-							</IconButton>
-						</Grid>
-					</SequenceRow>
-					<IntervalRow>
-						Step interval: <StepInput value={stepInterval} onChange={this.onChangeStepInterval} />
-						&nbsp;
-						<Select value={stepScale} onChange={this.onChangeStepScale} >
-							<MenuItem value={STEP_SCALE.HOUR}>hour(s)</MenuItem>
-							<MenuItem value={STEP_SCALE.DAY}>day(s)</MenuItem>
-							<MenuItem value={STEP_SCALE.MONTH}>month(s)</MenuItem>
-							<MenuItem value={STEP_SCALE.YEAR}>year(s)</MenuItem>
-							<MenuItem value={STEP_SCALE.FRAME}>frame(s)</MenuItem>
-						</Select>
-					</IntervalRow>
-					<SliderRow>
-						<Grid item>
-							<IconButton onClick={this.onClickPlayStop} ><this.PlayButtonIcon /></IconButton>
-						</Grid>
-						<Grid item>
-							<SequenceSlider
-								max={this.totalTime}
-								step={36000000}
-								value={this.currentTime}
-								onChange={(e, val) => this.goTo(val)}
-							/>
-						</Grid>
-					</SliderRow>
-					<FormGroup row>
-						<FormControlLabel
-							control={<Switch
-								checked={this.props.draggablePanels.includes(VIEWER_PANELS.LEGEND)}
-								onChange={this.props.toggleLegend}
-								color="primary"
-							/>}
-							label="Show Legend"
-						/>
-						<FormControlLabel
-							control={<Switch
-								checked={this.props.rightPanels.includes(VIEWER_PANELS.ACTIVITIES)}
-								onChange={this.props.toggleActivitiesPanel}
-								color="primary"
-							/>}
-							label="Full Activity List"
-						/>
-					</FormGroup>
-					{this.renderLoader(this.props.loadingFrame && !this.props.isActivitiesPending)}
-				</SequencePlayerColumn>
-			</SequencePlayerContainer>
-		);
-	}
-}
+/**
```

Comment:
very hard to read what's changed here... why does git thinks the whole file is modified? 😟

Reply:
@carmenfan I'm not sure. A few (not all) `tsx` files seemed a little odd, they'd load without any syntax highlighting and freeze my computer; this was one of them.

Reply:
can we try fixing it?

Reply:
@carmenfan It seems to be related to line endings. I thought git should take care of this, but it seems some `tsx` files have DOS-style line endings.

Reply:
ahh tsx needs to be added here https://github.com/3drepo/3drepo.io/blob/master/.gitattributes

Reply:
infact i don't know why it's not auto lf.

everywhere I've set has auto lf and do exclusion if necessary
https://github.com/3drepo/3drepobouncer/blob/master/.gitattributes